### PR TITLE
Update the JS page with info about CSS.

### PR DESCRIPTION
**Pull request by adiroiban**
Scope
=====

This is part 2 of our CSS and JS interaction.

Changes
=======

Update the JS page with info about CSS.

How to try and test the changes
===============================

reviewers: @ioanacristinamarinescu 

Check that changes make sense.

Thanks!

### CODE REVIEW

**Comment by adiroiban** [comment]
needs-review

File: docs/styleguide/css.rst [hunk 1]
```diff
@@ -90,8 +90,10 @@ CSS Class naming
 
   No CSS style should be defined on them.
 
-  No JS application functionality should be associated with them, other than the
-  testing code.
+  No JS application functionality should be associated with them, other than the testing code.
+
+  The testing code should use only CSS selectors defined based on these test CSS classed.
```
Comment (ioanacristinamarinescu) [inline docs/styleguide/css.rst:95]:
```suggestion
  The testing code should use only CSS selectors defined based on these test CSS classes.
```

File: docs/styleguide/javascript.rst [hunk 2]
```diff
@@ -298,70 +300,30 @@ private instances and create a new class scope.
...
-Now we know how to look for any JavaScript touching .js-open-tab, which should
-only be a simple search away.
-
-And hey, now JavaScript and CSS won't share selectors.
-Since we're separating our content and presentation, we might as well separate our behaviour all the
-way too.
-
-.. _slightly obtrusive javascript: http://ozmm.org/posts/slightly_obtrusive_javascript.html
-
+Use `test-CSS-CLASS-NAME` for any CSS classes that is used in the testing code.
```
Comment (ioanacristinamarinescu) [inline docs/styleguide/javascript.rst:309]:
```suggestion
Use `test-CSS-CLASS-NAME` for any CSS classes that are used in the testing code.
```

File: docs/styleguide/javascript.rst [hunk 3]
```diff
@@ -198,7 +199,11 @@ Example::
 
 
 TODO
-http://javascript.crockford.com/code.html http://jibbering.com/faq/notes/code-guidelines/ http://neil.rashbrook.org/Js.htm
```
Comment (ioanacristinamarinescu) [inline docs/styleguide/javascript.rst:201]: the last website does not work anymore. found it on archives https://web.archive.org/web/20160614115731/http://neil.rashbrook.org/Js.htm

**Review by ioanacristinamarinescu — APPROVED** [review]
Fixed some typos. Looks good.

File: docs/styleguide/javascript.rst (hunk 3 above)
Comment (adiroiban) [reply docs/styleguide/javascript.rst:201]: Thanks. yes. I also found it in the archive, but the content is not that important.

**Pull request merged**